torch/csrc/jit/ir/ir.h

```diff
@@ -326,6 +326,11 @@ struct TORCH_API Node {
   void setScope(ScopePtr scope) {
     scope_ = std::move(scope);
   }
+
+  void setModulePathStr(std::string modulePathStr) {
+    callstack_->get()->setModulePath(modulePathStr);
+  }
+
```

Comment:
I am vary of introducing extra methods to Node class if it can be avoided. I dont think we need it for this. See my other comments.